.kiro/steering/structure.md

```
@@ -1,105 +1,180 @@
-# Project Structure & Organization
+# Project Structure
 
-## Root Directory Structure
+## Root Organization
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)